app/src/main/java/com/ls/api/Processor.java

```diff
@@ -1,8 +1,51 @@
 package com.ls.api;
 
+import com.ls.drupalcon.model.data.Track;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Created by Varun Kumar on 7/27/2017.
  */
 
 public class Processor {
+    private String output;
+
+    public List<Track> listTrack = new ArrayList<>();
+
+
+    public Processor(String output) {
+        this.output = output;
+    }
+
+    public List<Track> trackProcessor()
+    {
+        try {
+            JSONArray tracks = new JSONArray(output);
+            int i, j;
+
+            Track track;
+
+            for(i=0;i<tracks.length();i++)
+            {
+                JSONObject tracksJSONObject = tracks.getJSONObject(i);
+
+                track = new Track();
+                track.setId(tracksJSONObject.getLong("id"));
+                track.setName(tracksJSONObject.getString("name"));
+
+                listTrack.add(track);
+            }
```

Comment:
@iamareebjamal I was looking not to use gson so I can have processing units for all the parts in one class unlike having serialisation tags in model to have similar structure with open event android app as it doesn't uses gson too. I'll remove that once all the managers are modified.